src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Same here - our entire app is being run through react at the route level so there's no need to bring it in for this individual component.